js/app.js

```diff
@@ -0,0 +1,60 @@
+//Section One
+//1. Interpolation occurs when new data is combined to produce new data
+const one = 1;
+const two = 2;
+console.log(one + two); 
+//should show an output of 3
+//2. Concatonation occurs when two strings are essentially smushed together
+const three = "three";
+const four = "four"
+console.log(three + four); 
+//should show an output of threefour
+
+//Section Two
+ const a = 4;
+ const b = 53;
+ const c = 57;
+ const d = 16;
+ const e = 'Kevin';
+
+ console.log(a < b);
+ console.log(c > d);
+ console.log('Name' === "Name");
+ console.log(a < b < c);
```

Comment:
what does this mean `a < b < c`?  it prints "true" but not for the reason you might think
the answer they were looking for is 
`a + b === c`